src/Build/BackEnd/Components/SdkResolution/DefaultSdkResolver.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.BackEnd
+{
+    /// <summary>
+    ///     Default SDK resolver for compatibility with VS2017 RTM.
+    /// </summary>
+    internal class DefaultSdkResolver : SdkResolver
+    {
+        public override string Name => "DefaultSdkResolver";
+
+        public override int Priority => 10000;
```

Comment:
We'll have to come up with something better for 3rd party support. Even if we do not expect many resolver implementations, there still will be some.